src/EditorFeatures/VisualBasicTest/ValidateFormatString/ValidateFormatStringTests.vb

```diff
@@ -0,0 +1,317 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.CodeFixes
+Imports Microsoft.CodeAnalysis.Diagnostics
+Imports Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.Diagnostics
+Imports Microsoft.CodeAnalysis.Options
+Imports Microsoft.CodeAnalysis.ValidateFormatString
+Imports Microsoft.CodeAnalysis.VisualBasic.ValidateFormatString
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.ValidateFormatString
+    Public Class ValidateFormatStringTests
+        Inherits AbstractVisualBasicDiagnosticProviderBasedUserDiagnosticTest
+
+        Friend Overrides Function CreateDiagnosticProviderAndFixer(
+                workspace As Workspace) As (DiagnosticAnalyzer, CodeFixProvider)
+            Return (New VisualBasicValidateFormatStringDiagnosticAnalyzer, Nothing)
+        End Function
+
+        Private Function VBOptionOnCSharpOptionOff() As IDictionary(Of OptionKey, Object)
+            Dim optionsSet = New Dictionary(Of OptionKey, Object) From
+            {
+                {New OptionKey(ValidateFormatStringOption.ReportInvalidPlaceholdersInStringDotFormatExpression, LanguageNames.CSharp), False},
+                {New OptionKey(ValidateFormatStringOption.ReportInvalidPlaceholdersInStringDotFormatExpression, LanguageNames.VisualBasic), True}
+            }
+
+            Return optionsSet
```

Comment:
fyi, there are help "Options" methods in the base class to make it easy to build up this dictionary